src/postgres-util/src/tunnel.rs

```diff
@@ -94,6 +94,10 @@ impl Default for TcpTimeoutConfig {
 }
 
 pub const DEFAULT_SNAPSHOT_STATEMENT_TIMEOUT: Duration = Duration::ZERO;
+// In PG16, this value defaults to 60 seconds, so we want to esnure we get at
```

Comment:
```suggestion
// In PG16, this value defaults to 60 seconds, so we want to ensure we get at
```

Reply:
ugh––going to live with this because it doesn't seem ambiguous enough to run back through CI.